databases/week3/MealApp_Tables/MealApp_Meal.sql

```diff
@@ -0,0 +1,57 @@
+-- MySQL dump 10.13  Distrib 8.0.21, for Linux (x86_64)
+--
+-- Host: localhost    Database: MealApp
+-- ------------------------------------------------------
+-- Server version	8.0.21
+
+/*!40101 SET @OLD_CHARACTER_SET_CLIENT=@@CHARACTER_SET_CLIENT */;
+/*!40101 SET @OLD_CHARACTER_SET_RESULTS=@@CHARACTER_SET_RESULTS */;
+/*!40101 SET @OLD_COLLATION_CONNECTION=@@COLLATION_CONNECTION */;
+/*!50503 SET NAMES utf8 */;
+/*!40103 SET @OLD_TIME_ZONE=@@TIME_ZONE */;
+/*!40103 SET TIME_ZONE='+00:00' */;
+/*!40014 SET @OLD_UNIQUE_CHECKS=@@UNIQUE_CHECKS, UNIQUE_CHECKS=0 */;
+/*!40014 SET @OLD_FOREIGN_KEY_CHECKS=@@FOREIGN_KEY_CHECKS, FOREIGN_KEY_CHECKS=0 */;
+/*!40101 SET @OLD_SQL_MODE=@@SQL_MODE, SQL_MODE='NO_AUTO_VALUE_ON_ZERO' */;
+/*!40111 SET @OLD_SQL_NOTES=@@SQL_NOTES, SQL_NOTES=0 */;
+
+--
+-- Table structure for table `Meal`
+--
+
+DROP TABLE IF EXISTS `Meal`;
+/*!40101 SET @saved_cs_client     = @@character_set_client */;
+/*!50503 SET character_set_client = utf8mb4 */;
+CREATE TABLE `Meal` (
+  `id` int NOT NULL AUTO_INCREMENT,
+  `title` varchar(255) DEFAULT NULL,
+  `description` text,
+  `location` varchar(255) DEFAULT NULL,
+  `when_create` datetime DEFAULT NULL,
+  `max_reservations` int DEFAULT NULL,
+  `price` decimal(10,0) DEFAULT NULL,
```

Comment:
If I wanted to price a certain meal at fx. **99.95** whould I be able to do that when using **DECIMAL(10,0)**? If not, what would need to change if I wanted to do this?

You could test if it is possible by using something like this:

`UPDATE Meal SET price = 99.95 WHERE id=3;`

Reply:
@arya56 here I'd suggest you have a look at [the MySQL manual on DECIMAL](https://dev.mysql.com/doc/refman/8.0/en/fixed-point-types.html).